apiserver/plane/db/models/asset.py

```diff
@@ -12,6 +12,7 @@
 
 
 def get_upload_path(instance, filename):
+    filename = filename[:50]
```

Comment:
Is this filename with extension or without? If with extension, I think we should preserve it.

Reply:
Yess!  @pablohashescobar, just as a note, I trim the image's filename while preserving the extension over [here](https://github.com/makeplane/plane/blob/c42bb5cca517805c5c55405aec61dad314d859e5/packages/editor/core/src/ui/plugins/image/image-upload-handler.ts#L132) while keeping the whole string length below 100.